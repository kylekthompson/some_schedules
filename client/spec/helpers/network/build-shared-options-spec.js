import buildSharedOptions from 'helpers/network/build-shared-options';

describe('buildSharedOptions', () => {
  it('returns the shared options', () => {
    expect(buildSharedOptions()).toMatchSnapshot();
  });
});
