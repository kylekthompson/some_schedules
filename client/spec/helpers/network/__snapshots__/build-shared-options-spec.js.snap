// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buildSharedOptions returns the shared options 1`] = `
Object {
  "credentials": "same-origin",
  "headers": Headers {
    "map": Object {
      "accept": "application/json",
      "content-type": "application/json",
    },
  },
}
`;
